cmd/server/main.go

```diff
@@ -114,6 +119,7 @@ func (o *ProxyRunOptions) Flags() *pflag.FlagSet {
 	flags.UintVar(&o.agentPort, "agent-port", o.agentPort, "Port we listen for agent connections on.")
 	flags.UintVar(&o.adminPort, "admin-port", o.adminPort, "Port we listen for admin connections on.")
 	flags.UintVar(&o.healthPort, "health-port", o.healthPort, "Port we listen for health connections on.")
+	flags.BoolVar(&o.enableContentionProfiling, "enable-contention-profiling", o.enableContentionProfiling, "enable contention profiling at host:admin-port/debug/pprof/debug")
```

Comment:
For me the pprof is at http://host:admin-port/debug/pprof

Reply:
Weird. For me it's debug/pprof/block, it's also the address in the [doc](https://golang.org/pkg/net/http/pprof/). debug/block returns 404 for me. Can you double check?

Reply:
Oh I see /debug/pprof/debug is a typo. I fixed it.

Reply:
Looks like we both typo'd that one :smile_cat: